src/Simple.OData.Client.UnitTests/Core/ExpansionTests.cs

```diff
@@ -331,7 +331,7 @@ public async Task ExpandCategorySelectProductNameCategoryName(string metadataFil
 
         [Theory]
         [InlineData("Northwind3.xml", "Products?$expand=*&$select=ProductName,Category/CategoryName")]
-        [InlineData("Northwind4.xml", "Products?$expand=*($select=CategoryName)&$select=ProductName")]
+        [InlineData("Northwind4.xml", "Products?$expand=*&$select=ProductName")]
```

Comment:
Is that expand+select syntax no longer supported? And if so, is that a breaking change we'll need to communicate?

Reply:
Expand+select is supported but only for concrete navigation properties e.g. $expand=Category($select=CategoryName). According to OData v4 specification (at least as implemented by Microsoft) a star operator (expand all navigation properties) can be combined only with $levels clause for cyclic properties. Because of star operator the server response will include Category/CategoryName.